docs/configuration/index-config.md

```diff
@@ -217,10 +218,11 @@ precision: milliseconds
 | Variable      | Description   | Default value |
 | ------------- | ------------- | ------------- |
 | `input_formats` | Formats used to parse input dates | [`rfc3339`, `unix_timestamp`] |
+| `output_format` | Format used to display dates in search results | `rfc3339` |
 | `stored`        | Whether the field values are stored in the document store | `true` |
 | `indexed`       | Whether the field values are indexed | `true` |
 | `fast`          | Whether the field values are stored in a fast field | `false` |
-| `precision`     | The precision (`seconds`, `milliseconds`, or `microseconds`) used to store the fast values | `seconds` |
+| `precision`     | The precision (`seconds`, `milliseconds`, or `microseconds`) used to store the fast values. It's also used to display date when output_format is set to `unix_timestamp`  | `seconds` |
```

Comment:
It would be nice to explain in the docs that:
- values are stored in the term dictionary at seconds precision
- values are stored in fast field storage  at the given precision
- values are stored in the doc store as is.

Reply:
So my concern about adding this in the docs is that they are implementation details. I wanted to just limit to what precision is used for. The rest as long as we return correctly what use put in.

Reply:
Yes, you're right. You can add maybe that in the `docs/internals` :)